<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="addInstantDose">
     <property name="styleSheet">
      <string notr="true">background:#FFFFFF;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QGridLayout" name="gridLayout_20">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="horizontalSpacing">
         <number>20</number>
        </property>
        <item row="0" column="0">
         <widget class="QGroupBox" name="FAB_8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Nirmala UI</family>
            <pointsize>15</pointsize>
            <weight>87</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox{
border:1px solid lightgrey;
font-weight: 700;
padding:10px 0px 0px 0px;
margin-top:12px;
font-family:Nirmala UI;
font-size:15pt;
border-radius:10%
}
QGroupBox:title{
subcontrol-origin: margin;
left: 10px;
top:-4px;
margin:0px 2px 0px 2px
}
</string>
          </property>
          <property name="title">
           <string>Left Pump</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_37">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>9</number>
           </property>
           <property name="topMargin">
            <number>9</number>
           </property>
           <property name="rightMargin">
            <number>9</number>
           </property>
           <property name="bottomMargin">
            <number>9</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_40">
               <property name="spacing">
                <number>2</number>
               </property>
               <item alignment="Qt::AlignLeft">
                <widget class="QLabel" name="drugNameLeftInstantDose">
                 <property name="styleSheet">
                  <string notr="true">font-weight:700;
font-size:12pt</string>
                 </property>
                 <property name="text">
                  <string>---</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_12">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_21">
               <property name="spacing">
                <number>2</number>
               </property>
               <item alignment="Qt::AlignLeft">
                <widget class="QLabel" name="label_28">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <weight>87</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font-weight:700;
font-size:12pt</string>
                 </property>
                 <property name="text">
                  <string>Dose amount</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txtDoseLeft">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>Nirmala UI</family>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>Enter dose amount</string>
                 </property>
                 <property name="clearButtonEnabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="errDoseLeft">
                 <property name="styleSheet">
                  <string notr="true">color:red</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_22">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QProgressBar" name="progressBarLeft">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border:None;</string>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="textVisible">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_18">
               <property name="spacing">
                <number>10</number>
               </property>
               <property name="topMargin">
                <number>35</number>
               </property>
               <property name="rightMargin">
                <number>9</number>
               </property>
               <item alignment="Qt::AlignHCenter">
                <widget class="QPushButton" name="btnFillLeft">
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::TabFocus</enum>
                 </property>
                 <property name="autoFillBackground">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Fill</string>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0" colspan="2" alignment="Qt::AlignRight|Qt::AlignBottom">
         <widget class="QLabel" name="infoInstantFill">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="FAB_9">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Nirmala UI</family>
            <pointsize>15</pointsize>
            <weight>87</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox{
border:1px solid lightgrey;
font-weight: 700;
padding:10px 0px 0px 0px;
margin-top:12px;
font-family:Nirmala UI;
font-size:15pt;
border-radius:10%
}
QGroupBox:title{
subcontrol-origin: margin;
left: 10px;
top:-4px;
margin:0px 2px 0px 2px
}
</string>
          </property>
          <property name="title">
           <string>Right Pump</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_38">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>9</number>
           </property>
           <property name="topMargin">
            <number>9</number>
           </property>
           <property name="rightMargin">
            <number>9</number>
           </property>
           <property name="bottomMargin">
            <number>9</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_37">
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_41">
               <property name="spacing">
                <number>2</number>
               </property>
               <item alignment="Qt::AlignLeft">
                <widget class="QLabel" name="drugNameRightInstantDose">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <weight>87</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font-weight:700;
font-size:12pt</string>
                 </property>
                 <property name="text">
                  <string>---</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_13">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_40">
               <property name="spacing">
                <number>2</number>
               </property>
               <item alignment="Qt::AlignLeft">
                <widget class="QLabel" name="label_50">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <weight>87</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font-weight:700;
font-size:12pt</string>
                 </property>
                 <property name="text">
                  <string>Dose amount</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txtDoseRight">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>Nirmala UI</family>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>Enter dose amount</string>
                 </property>
                 <property name="clearButtonEnabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="errDoseRight">
                 <property name="styleSheet">
                  <string notr="true">color:red</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_41">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QProgressBar" name="progressBarRight">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border:none;</string>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="textVisible">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_39">
               <property name="spacing">
                <number>10</number>
               </property>
               <property name="topMargin">
                <number>35</number>
               </property>
               <property name="rightMargin">
                <number>9</number>
               </property>
               <item alignment="Qt::AlignHCenter">
                <widget class="QPushButton" name="btnFillRight">
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::TabFocus</enum>
                 </property>
                 <property name="autoFillBackground">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Fill</string>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
